protocol/contracts/v2/TreasuryReservesVault.sol

```diff
@@ -183,10 +183,39 @@ contract TreasuryReservesVault is ITreasuryReservesVault, TempleElevatedAccess {
 
         ITempleStrategy.AssetBalance memory _assetBalance;
         uint256 _length = debtCeiling.length;
+        IERC20 _token;
         for (uint256 i; i < _length; ++i) {
             _assetBalance = debtCeiling[i];
-            if (!_borrowTokenSet.contains(address(_assetBalance.asset))) revert BorrowTokenNotEnabled();
-            strategyConfig.debtCeiling[IERC20(_assetBalance.asset)] = _assetBalance.balance;
+            _token = IERC20(_assetBalance.asset);
+            if (!_borrowTokenSet.contains(address(_token))) revert BorrowTokenNotEnabled();
+            strategyConfig.debtCeiling[_token] = _assetBalance.balance;
+            strategyConfig.enabledBorrowTokens[_token] = true;
+        }
+    }
+
+    /**
+     * @notice Enable and/or disable tokens which a strategy can borrow from the (configured) TRV borrow tokens
+     */
+    function updateStrategyEnabledBorrowTokens(
+        address strategy, 
+        IERC20[] calldata enableBorrowTokens, 
+        IERC20[] calldata disableBorrowTokens
+    ) external override onlyElevatedAccess {
+        StrategyConfig storage _strategyConfig = _getStrategyConfig(strategy);
+
+        uint256 i;
+        IERC20 token;
+        uint256 _length = enableBorrowTokens.length;
+        for (; i < _length; ++i) {
+            token = enableBorrowTokens[i];
+            if (!_borrowTokenSet.contains(address(token))) revert BorrowTokenNotEnabled();
+            _strategyConfig.enabledBorrowTokens[token] = true;
+        }
+
+        _length = disableBorrowTokens.length;
+        for (i = 0; i < _length; ++i) {
```

Comment:
nit: `if (_length > 0)`

Reply:
It's the same result though right? If `_length == 0`, then the condition on the for loop won't be met so it'll just skip.

I'm not sure it's worth the extra bytecode to wrap in another if statement imo